src/lib/core/components/visualizations/Visualizations.scss

```
@@ -110,6 +110,7 @@
       color: #444;
       font-weight: 500;
       text-align: center;
+      font-size: 16px;
```

Could you change this to use `em` units? They are proportional (whereas `px` is fixed), which makes it easier to scale entire sections of the page, if needed. This is a **big** area of debate. Here is a [good introduction to the topic](https://engageinteractive.co.uk/blog/em-vs-rem-vs-px), if you're interested :slightly_smiling_face: .
```suggestion
      font-size: 1.2em;
```

---

Done! Oh fascinating. Thank you for the article it was really helpful!